sdk/digitaltwins/Azure.DigitalTwins.Core/samples/Readme.md

```diff
@@ -297,16 +296,38 @@ AsyncPageable<BasicDigitalTwin> asyncPageableResponseQueryBuilder = client.Query
 
 ### Build ADT Queries
 
-Build an [Azure DigitalTwins Query Store Language](https://docs.microsoft.com/azure/digital-twins/concepts-query-language) using an `DigitalTwinsQueryBuilder`. When using a `Where` clause, conditions are separated from the `Where` keyword (see `whereIsOfModel`)
+Build an [Azure DigitalTwins Query](https://docs.microsoft.com/azure/digital-twins/concepts-query-language) using a `DigitalTwinsQueryBuilder`. Two different objects entitled `DigitalTwinsQueryBuilder` exist in two namespaces, each with their own unique functionality and features. We encourage testing both objects and determining which works best given a specific set of needs.
+
+```C#
+// The common namespace for DigitalTwins query builders
+// Must be used regardless of which type of query builder is in usage
+using Azure.DigitalTwins.Core.QueryBuilder;
+
+// Fluent-style DigitalTwins query builder
+using Azure.DigitalTwins.Core.QueryBuilder.Fluent;
```

Comment:
Don't we need to alias these to avoid naming conflicts?

Reply:
We do where we have to. The samples of the two different approaches live in two different files so no need for aliasing in those cases.